src/test/run-pass/self-struct-ctor.rs

```diff
@@ -0,0 +1,137 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Copyright no longer necessary.

Reply:
Thanks!  Copyright is removed. And the test is moved to `test/ui/run-pass/rfcs`.